pandas/core/panel.py

```diff
@@ -165,6 +165,13 @@ def _init_data(self, data, copy, dtype, **kwargs):
             mgr = self._init_matrix(data, passed_axes, dtype=dtype, copy=copy)
             copy = False
             dtype = None
+        elif lib.isscalar(data) and all(x is not None for x in passed_axes):
+            if dtype is None:
+                dtype, data = _infer_dtype_from_scalar(data)
+            values = np.empty([len(x) for x in passed_axes], dtype=dtype)
+            values.fill(data)
+            mgr = self._init_matrix(values, passed_axes, dtype=dtype, copy=copy)
```

Comment:
I think you need to make ``copy= False`` before you init the matrix (so it doesn't copy it again).